Update to MP6.1

gkwan-ibm opened the pull request:
address issue: https://github.com/OpenLiberty/guides-common/issues/1021

### Review changes
- [x] pom.xml
  - microfile-profile is 6.1, and dependencies
  - LMP 3.10
  - `default.` should be removed
  - ports `9090`,`9453`,`9091`,`9454` if guide uses local kube
- [x] server.xml
  - feature version
- [x] index.html
  - feature version
- [x] update lgdev with `version-update-mp61` branch
- [x] do end-to-end test and review content carefully
  - clone the `version-update-mp61` branch
  - if index.html has changes, make sure the links work

gkwan-ibm committed:
update versions

gkwan-ibm committed:
Update test.yml

gkwan-ibm committed:
Update docker-compose.yml

gkwan-ibm committed:
Update dailyBuild.sh

gkwan-ibm committed:
Update README.adoc

gkwan-ibm committed:
Update README.adoc

gkwan-ibm committed:
Update testApp.sh

gkwan-ibm committed:
update versions

Ruilin-Ma committed:
change port setting for system service to avoid conflict with Rancher desktop

Ruilin-Ma committed:
change port 9080 -> 9090

Ruilin-Ma committed:
fixing hotspot display issue

Ruilin-Ma committed:
fixing hotspot display issue

Ruilin-Ma reviewed (dismissed):
Tested the guide end to end on Mac, Windows, and Linux VM, no issues found

gkwan-ibm committed:
update inventory service ports

gkwan-ibm committed:
Merge branch 'staging' into version-update-mp61

gkwan-ibm committed:
Update dependabot.yml

gkwan-ibm committed:
Update add-pr-to-project.yml

gkwan-ibm committed:
Merge branch 'staging' into version-update-mp61

gkwan-ibm committed:
Merge branch 'staging' into version-update-mp61

gkwan-ibm committed:
Dependabot bump (#102)

* Bump org.apache.maven.plugins:maven-failsafe-plugin in /start/system

Bumps [org.apache.maven.plugins:maven-failsafe-plugin](https://github.com/apache/maven-surefire) from 3.0.0 to 3.2.5.
- [Release notes](https://github.com/apache/maven-surefire/releases)
- [Commits](https://github.com/apache/maven-surefire/compare/surefire-3.0.0...surefire-3.2.5)

---
updated-dependencies:
- dependency-name: org.apache.maven.plugins:maven-failsafe-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.slf4j:slf4j-simple from 2.0.7 to 2.0.12 in /start/system

Bumps org.slf4j:slf4j-simple from 2.0.7 to 2.0.12.

---
updated-dependencies:
- dependency-name: org.slf4j:slf4j-simple
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump io.openliberty.tools:liberty-maven-plugin in /start/inventory

Bumps [io.openliberty.tools:liberty-maven-plugin](https://github.com/OpenLiberty/ci.maven) from 3.8.2 to 3.10.1.
- [Release notes](https://github.com/OpenLiberty/ci.maven/releases)
- [Commits](https://github.com/OpenLiberty/ci.maven/compare/liberty-maven-3.8.2...liberty-maven-3.10.1)

---
updated-dependencies:
- dependency-name: io.openliberty.tools:liberty-maven-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.slf4j:slf4j-simple from 2.0.7 to 2.0.12 in /start/inventory

Bumps org.slf4j:slf4j-simple from 2.0.7 to 2.0.12.

---
updated-dependencies:
- dependency-name: org.slf4j:slf4j-simple
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.jboss.resteasy:resteasy-client in /finish/system

Bumps [org.jboss.resteasy:resteasy-client](https://github.com/resteasy/resteasy) from 6.2.3.Final to 6.2.7.Final.
- [Release notes](https://github.com/resteasy/resteasy/releases)
- [Commits](https://github.com/resteasy/resteasy/compare/6.2.3.Final...6.2.7.Final)

---
updated-dependencies:
- dependency-name: org.jboss.resteasy:resteasy-client
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.eclipse.microprofile:microprofile in /start/system

Bumps [org.eclipse.microprofile:microprofile](https://github.com/eclipse/microprofile) from 6.0 to 6.1.
- [Release notes](https://github.com/eclipse/microprofile/releases)
- [Commits](https://github.com/eclipse/microprofile/compare/6.0...6.1)

---
updated-dependencies:
- dependency-name: org.eclipse.microprofile:microprofile
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.apache.maven.plugins:maven-war-plugin in /start/inventory

Bumps [org.apache.maven.plugins:maven-war-plugin](https://github.com/apache/maven-war-plugin) from 3.3.2 to 3.4.0.
- [Commits](https://github.com/apache/maven-war-plugin/compare/maven-war-plugin-3.3.2...maven-war-plugin-3.4.0)

---
updated-dependencies:
- dependency-name: org.apache.maven.plugins:maven-war-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.apache.maven.plugins:maven-failsafe-plugin in /finish/system

Bumps [org.apache.maven.plugins:maven-failsafe-plugin](https://github.com/apache/maven-surefire) from 3.0.0 to 3.2.5.
- [Release notes](https://github.com/apache/maven-surefire/releases)
- [Commits](https://github.com/apache/maven-surefire/compare/surefire-3.0.0...surefire-3.2.5)

---
updated-dependencies:
- dependency-name: org.apache.maven.plugins:maven-failsafe-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.eclipse.microprofile:microprofile in /finish/inventory

Bumps [org.eclipse.microprofile:microprofile](https://github.com/eclipse/microprofile) from 6.0 to 6.1.
- [Release notes](https://github.com/eclipse/microprofile/releases)
- [Commits](https://github.com/eclipse/microprofile/compare/6.0...6.1)

---
updated-dependencies:
- dependency-name: org.eclipse.microprofile:microprofile
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump au.com.dius.pact.provider:junit5 in /finish/system

Bumps [au.com.dius.pact.provider:junit5](https://github.com/pact-foundation/pact-jvm) from 4.5.6 to 4.6.7.
- [Release notes](https://github.com/pact-foundation/pact-jvm/releases)
- [Changelog](https://github.com/pact-foundation/pact-jvm/blob/master/CHANGELOG.md)
- [Commits](https://github.com/pact-foundation/pact-jvm/commits)

---
updated-dependencies:
- dependency-name: au.com.dius.pact.provider:junit5
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.jboss.resteasy:resteasy-client in /start/system

Bumps [org.jboss.resteasy:resteasy-client](https://github.com/resteasy/resteasy) from 6.2.3.Final to 6.2.7.Final.
- [Release notes](https://github.com/resteasy/resteasy/releases)
- [Commits](https://github.com/resteasy/resteasy/compare/6.2.3.Final...6.2.7.Final)

---
updated-dependencies:
- dependency-name: org.jboss.resteasy:resteasy-client
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.apache.maven.plugins:maven-war-plugin in /finish/system

Bumps [org.apache.maven.plugins:maven-war-plugin](https://github.com/apache/maven-war-plugin) from 3.3.2 to 3.4.0.
- [Commits](https://github.com/apache/maven-war-plugin/compare/maven-war-plugin-3.3.2...maven-war-plugin-3.4.0)

---
updated-dependencies:
- dependency-name: org.apache.maven.plugins:maven-war-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.eclipse.microprofile:microprofile in /start/inventory

Bumps [org.eclipse.microprofile:microprofile](https://github.com/eclipse/microprofile) from 6.0 to 6.1.
- [Release notes](https://github.com/eclipse/microprofile/releases)
- [Commits](https://github.com/eclipse/microprofile/compare/6.0...6.1)

---
updated-dependencies:
- dependency-name: org.eclipse.microprofile:microprofile
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump io.openliberty.tools:liberty-maven-plugin in /finish/inventory

Bumps [io.openliberty.tools:liberty-maven-plugin](https://github.com/OpenLiberty/ci.maven) from 3.8.2 to 3.10.1.
- [Release notes](https://github.com/OpenLiberty/ci.maven/releases)
- [Commits](https://github.com/OpenLiberty/ci.maven/compare/liberty-maven-3.8.2...liberty-maven-3.10.1)

---
updated-dependencies:
- dependency-name: io.openliberty.tools:liberty-maven-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump io.openliberty.tools:liberty-maven-plugin in /start/system

Bumps [io.openliberty.tools:liberty-maven-plugin](https://github.com/OpenLiberty/ci.maven) from 3.8.2 to 3.10.1.
- [Release notes](https://github.com/OpenLiberty/ci.maven/releases)
- [Commits](https://github.com/OpenLiberty/ci.maven/compare/liberty-maven-3.8.2...liberty-maven-3.10.1)

---
updated-dependencies:
- dependency-name: io.openliberty.tools:liberty-maven-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump io.openliberty.tools:liberty-maven-plugin in /finish/system

Bumps [io.openliberty.tools:liberty-maven-plugin](https://github.com/OpenLiberty/ci.maven) from 3.8.2 to 3.10.1.
- [Release notes](https://github.com/OpenLiberty/ci.maven/releases)
- [Commits](https://github.com/OpenLiberty/ci.maven/compare/liberty-maven-3.8.2...liberty-maven-3.10.1)

---
updated-dependencies:
- dependency-name: io.openliberty.tools:liberty-maven-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.apache.maven.plugins:maven-war-plugin in /start/system

Bumps [org.apache.maven.plugins:maven-war-plugin](https://github.com/apache/maven-war-plugin) from 3.3.2 to 3.4.0.
- [Commits](https://github.com/apache/maven-war-plugin/compare/maven-war-plugin-3.3.2...maven-war-plugin-3.4.0)

---
updated-dependencies:
- dependency-name: org.apache.maven.plugins:maven-war-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.jboss.resteasy:resteasy-client in /start/inventory

Bumps [org.jboss.resteasy:resteasy-client](https://github.com/resteasy/resteasy) from 6.2.3.Final to 6.2.7.Final.
- [Release notes](https://github.com/resteasy/resteasy/releases)
- [Commits](https://github.com/resteasy/resteasy/compare/6.2.3.Final...6.2.7.Final)

---
updated-dependencies:
- dependency-name: org.jboss.resteasy:resteasy-client
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.apache.maven.plugins:maven-war-plugin in /finish/inventory

Bumps [org.apache.maven.plugins:maven-war-plugin](https://github.com/apache/maven-war-plugin) from 3.3.2 to 3.4.0.
- [Commits](https://github.com/apache/maven-war-plugin/compare/maven-war-plugin-3.3.2...maven-war-plugin-3.4.0)

---
updated-dependencies:
- dependency-name: org.apache.maven.plugins:maven-war-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.eclipse.microprofile:microprofile in /finish/system

Bumps [org.eclipse.microprofile:microprofile](https://github.com/eclipse/microprofile) from 6.0 to 6.1.
- [Release notes](https://github.com/eclipse/microprofile/releases)
- [Commits](https://github.com/eclipse/microprofile/compare/6.0...6.1)

---
updated-dependencies:
- dependency-name: org.eclipse.microprofile:microprofile
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.slf4j:slf4j-simple from 2.0.7 to 2.0.12 in /finish/system

Bumps org.slf4j:slf4j-simple from 2.0.7 to 2.0.12.

---
updated-dependencies:
- dependency-name: org.slf4j:slf4j-simple
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump au.com.dius.pact.provider:maven in /finish/inventory

Bumps [au.com.dius.pact.provider:maven](https://github.com/pact-foundation/pact-jvm) from 4.5.6 to 4.6.7.
- [Release notes](https://github.com/pact-foundation/pact-jvm/releases)
- [Changelog](https://github.com/pact-foundation/pact-jvm/blob/master/CHANGELOG.md)
- [Commits](https://github.com/pact-foundation/pact-jvm/commits)

---
updated-dependencies:
- dependency-name: au.com.dius.pact.provider:maven
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.slf4j:slf4j-simple from 2.0.7 to 2.0.12 in /finish/inventory

Bumps org.slf4j:slf4j-simple from 2.0.7 to 2.0.12.

---
updated-dependencies:
- dependency-name: org.slf4j:slf4j-simple
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.jboss.resteasy:resteasy-client in /finish/inventory

Bumps [org.jboss.resteasy:resteasy-client](https://github.com/resteasy/resteasy) from 6.2.3.Final to 6.2.7.Final.
- [Release notes](https://github.com/resteasy/resteasy/releases)
- [Commits](https://github.com/resteasy/resteasy/compare/6.2.3.Final...6.2.7.Final)

---
updated-dependencies:
- dependency-name: org.jboss.resteasy:resteasy-client
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

* Bump org.apache.maven.plugins:maven-failsafe-plugin in /finish/inventory

Bumps [org.apache.maven.plugins:maven-failsafe-plugin](https://github.com/apache/maven-surefire) from 3.0.0 to 3.2.5.
- [Release notes](https://github.com/apache/maven-surefire/releases)
- [Commits](https://github.com/apache/maven-surefire/compare/surefire-3.0.0...surefire-3.2.5)

---
updated-dependencies:
- dependency-name: org.apache.maven.plugins:maven-failsafe-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

---------

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

gkwan-ibm committed:
Update SystemResource.java

gkwan-ibm committed:
Update pom.xml

gkwan-ibm committed:
Update pom.xml

gkwan-ibm committed:
Update pom.xml

gkwan-ibm committed:
Update pom.xml

gkwan-ibm committed:
Update docker-compose.yml

gkwan-ibm committed:
Update README.adoc

gkwan-ibm committed:
Update testApp.sh

gkwan-ibm committed:
Update README.adoc

gkwan-ibm committed:
Update docker-compose.yml

gkwan-ibm committed:
Update testApp.sh

gkwan-ibm committed:
Update README.adoc

Ruilin-Ma reviewed (approved):
Tested the guide end to end with Java 17 on Mac, Win, and Linux with no issue found
